### PR TITLE
Drop YAML packaging

### PR DESCRIPTION
**Pull request by martyone**

### CODE REVIEW

File: rpm/acpid.spec [hunk 1]
```diff
@@ -28,13 +20,13 @@ Source9:    acpid-start-script
...
 Source11:    acpid.service
 Source12:    acpid.conf
-Source100:  acpid.yaml
 Patch0:     acpid-2.0.9-makefile.patch
 Patch1:     acpid-2.0.14-bugfix-incorrect-sizeof-usage-for-memset.patch
 Requires:   systemd
 Requires(preun): systemd
 Requires(post): systemd
 Requires(postun): systemd
+BuildRequires:  pkgconfig(systemd)
```
Comment (rainemak) [inline rpm/acpid.spec:29]: Why is this needed?

Comment (martyone) [reply rpm/acpid.spec:29]: Macros like %{_unitdir} are provided by the systemd package.

**Review by rainemak — APPROVED** [review]
